test/index.test.js

```diff
@@ -1123,3 +1123,79 @@ describe('FastifyInstrumentation', () => {
     })
   })
 })
+
+// OTEL_SERVICE_NAME
+// https://opentelemetry.io/docs/languages/sdk-configuration/general/
+describe('FastifyInstrumentation with General SDK Configuration set', () => {
+  process.env.OTEL_SERVICE_NAME = 'my_app';
```

Comment:
I would move this block in a new file (`env-vars.test.js`) so we can improve the env compatibility and add before/after to manage the `process.env` without interfering with other tests